daemon/internal/newrelic/integration/test.go

```diff
@@ -823,7 +815,7 @@ func (t *Test) comparePhpPackages(harvest *newrelic.Harvest) {
 		}
 
 		// create notes for all packages in the actual list not in the expected list
-		for ii, _ := range actualPackages {
+		for ii := range actualPackages {
```

Comment:
why not just use 
`for ii, aPkg:= range actualPackages {`

Then refernece aPgk directly later [here](https://github.com/newrelic/newrelic-php-agent/pull/1103/files#diff-c7f1af7665ab0865f15e79eeba7e73a7295fe66e0ff9205b9781c7b0d7c75faeR821) and [here](https://github.com/newrelic/newrelic-php-agent/pull/1103/files#diff-c7f1af7665ab0865f15e79eeba7e73a7295fe66e0ff9205b9781c7b0d7c75faeR827)?

Reply:
Changing this logic is outside the scope of this PR. This was an auto-change generated by Go's formatter.